apps/web/screens/ServiceWeb/SubPage/SubPage.tsx

```diff
@@ -204,10 +224,63 @@ const SubPage: Screen<SubPageProps> = ({
                         <Text variant="h2" as="h2">
                           {question.title}
                         </Text>
-
                         <Box>
                           {richText(question.answer as SliceType[], undefined)}
                         </Box>
+                        <>
+                          {question.relatedLinks?.length > 0 && (
+                            <Box
+                              background="purple100"
+                              borderRadius="large"
+                              padding={4}
+                              marginTop={6}
+                              marginBottom={2}
+                            >
+                              <Text fontWeight="semiBold">
+                                {o(
+                                  'serviceWebRelatedMaterialHeaderTitle',
+                                  'Tengt efni',
+                                )}
+                              </Text>
+                              <GridColumn>
+                                {(question.relatedLinks ?? []).map(
+                                  ({ text, url }, index) => (
+                                    <GridRow marginTop={2}>
+                                      <Link
+                                        underline="small"
+                                        underlineVisibility="hover"
+                                        href={url}
+                                        key={index}
+                                      >
+                                        {text}
+                                      </Link>
+                                    </GridRow>
+                                  ),
+                                )}
+                              </GridColumn>
+                            </Box>
+                          )}
+                          {institutionSlugBelongsToMannaudstorg && (
+                            <Box
+                              marginTop={
+                                question.relatedLinks?.length > 0 ? 0 : 4
+                              }
+                            >
+                              <OrganizationContactBanner
+                                organizationLogoUrl={organization.logo?.url}
+                                contactLink="https://gatt.fjs.is/plugins/servlet/desk/portal/11/create/127"
```

Comment:
Can this not be stored somehow?
If this becomes obsolete a programmer will have to manually update it.

Reply:
This link will be different depending on what support qna you are looking at, I know of the technical debt and I will revisit this at a later point when it is known how the links will be implemented.